tests/test_pipeline/test_pipeline.py

```diff
@@ -1128,6 +1216,13 @@ def test_forecast_backtest_correct_ordering(step_ts: TSDataset):
     assert np.all(forecast_df.values == expected_forecast_df.values)
 
 
+def test_get_historical_forecasts_correct_ordering(step_ts: TSDataset):
```

Comment:
Is this test really about ordering? I can see comparing raw values.

Reply:
Same naming as in the test above for `backtest` to show it is the similar test.